WordPress/gutenberg
# [RNMobile] Block Inserter Search: Update platform specific styles

## Description
This updates the styling of the Inserter Block Search for mobile.

## How has this been tested?
- Open the inserter on mobile
    - Verify that the inactive search state looks correct
- Try typing a few characters
   -  Verify that the active search state looks correct
  

## Screenshots 

## Types of changes
- Converts CSS selectors to use the BEM conventions
- Add platform specific styling 

## Checklist:
